ppr-ui/src/components/registration/RegistrationBar.vue

```diff
@@ -206,34 +207,13 @@ export default defineComponent({
   font-weight: normal;
   box-shadow: none;
 }
-::v-deep .actions__more-actions__btn {
+.actions__more-actions__btn {
   width: 40;
   border-top-left-radius: 0;
   border-bottom-left-radius: 0;
   box-shadow: none;
 }
-
-#search-btn-info {
-  color: $gray8;
-  font-size: 0.725rem;
-}
-.registration-info {
-  color: $gray8;
-  font-size: 0.875rem;
-}
-.registration-title {
-  color: $gray9;
-  font-size: 1rem;
-}
-.registration-select {
-  background-color: $BCgovBlue3;
-  color: white;
-}
-::v-deep .v-list-item .v-list-item__title {
-  font-size: 0.875rem;
-  color: $gray9;
-}
-::v-deep .v-list-item:hover {
+:v-deep .v-list-item:hover {
```

Comment:
this works?? Ah I wish my past self new this

Reply:
lol, it doesn't good catch. i popped in one more change.

Reply:
oh okay haha